versioned_docs/version-3.0/ios-quickstart-manual.md

```diff
@@ -0,0 +1,325 @@
+---
+title: "Quickstart for integrating purchases with Adapty into your custom paywall in iOS SDK"
+description: "Quickstart guide to setting up Adapty for in-app subscription management."
+metadataTitle: "Adapty Quickstart Guide | Adapty Docs"
+keywords: ['paywalls ios', 'sdk ios', 'paywall', 'getPaywall']
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This guide shows you how to integrate Adapty's purchase handling into your custom paywall UI. You'll fetch products from Adapty, process purchases, and handle restores, while maintaining full control over your paywall design.
+
+:::important
+**This guide is for implementing custom paywalls.** If you want the easiest way to enable purchases, use the [Adapty Paywall Builder](ios-quickstart-paywalls.md). With Paywall Builder, you create paywalls in a no-code visual editor, and Adapty handles all purchase logic automatically.
+:::
+
+## Before you start
+
+### Set up products
+
+To enable in-app purchases, you need to understand three key concepts:
+
+- [**Products**](product.md) – anything users can buy (subscriptions, consumables, lifetime access)
+- [**Paywalls**](paywalls.md) are configurations that define which products to offer. In Adapty, paywalls are the only way to retrieve products, but this design lets you modify offerings, pricing, and product combinations without touching your app code.
+- [**Placements**](placements.md) – where and when you show paywalls in your app (like `main`, `onboarding`, `settings`). You set up paywalls for placements in the dashboard, then request them by placement ID in your code. This makes it easy to run A/B tests and show different paywalls to different users.
+
+Working with these concepts is required even if you work with your custom paywall. Basically, they are just your way to manage the products you sell in your app.
+
+When implementing your custom paywall, you will need to use a **placement ID** to retrieve products, so follow the quickstart guide [here](quickstart.md).
+
+### Manage users
+
+You can work either with or without backend authentication on your side.
+
+However, the Adapty SDK handles anonymous and identified users differently. Read the [identification quickstart guide](ios-quickstart-identify.md) to understand the specifics and ensure you are working with users properly.
+
+## Step 1. Get products
+
+To retrieve products for your custom paywall, you need to:
+
+1. Get the `paywall` object by the [placement](placements.md) ID using the `getPaywall` method.
```

Comment:
1) No need to use "using" like that
2) "by" and "using" in the same sentence is too much. We can split it into two shorter sentences if need be
3) in the expression "get X by Y", the definite article is unnecessary (e.g. "find person by name")
4) Perhaps we need to be more specific? 

```suggestion
1. Get the `paywall` object from Adapty. Pass a [placement](placements.md) ID to the `getPaywall` method. 
```